media/js/mozorg/home/fundraising-nav-cta.es6.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrafficCop from '@mozmeao/trafficcop';
+import { isApprovedToRun } from '../../base/experiment-utils.es6.js';
+
+const href = window.location.href;
+
+const init = () => {
+    if (href.indexOf('v=1') !== -1) {
+        window.dataLayer.push({
+            'data-ex-variant': 'fundraising-nav-cta-v1',
+            'data-ex-name': 'fundraising-nav-cta-july2024'
+        });
```

Comment:
**issue:** these are GA3 events, which is no longer used in production. Instead we should use the newer GA4 event which is in the docs here:

https://bedrock.readthedocs.io/en/latest/abtest.html#recording-the-data

Reply:
Just to confirm:
- `data-ex-name` should be changed to `event`
- `data-ex-variant` should be changed to `variant`?

Reply:
See the GA4 examples I linked to above e.g

```
window.dataLayer.push({
        event: 'experiment_view',
        id: 'Berlin-Campaign-Landing-Page',
        variant: 'de-page',
});
```